### DISCUSSION TITLE
Correct Renovate annotation for Maven install-tool

### DISCUSSION THREAD

**Discussion by kennethjor** [renovatebot/renovate#16135] [Request Help] [answered]
I have a Docker image based on Containerbase which installs Maven like this:
```
# renovate: datasource=maven versioning=maven
RUN install-tool maven 3.6.3
```
I recently ran Renovate over this and it did not create a PR for 3.8.6 update. AFAIK my [renovate.json](https://github.com/autonomouslogic/semantic-release-docker/blob/main/renovate.json#L39) is correct for this type of stuff. What would be the correct datasource and versioning for this?

https://github.com/autonomouslogic/semantic-release-docker/blob/main/Dockerfile#L30

**Comment by viceice** [comment, marked as answer]
https://github.com/containerbase/buildpack/blob/main/test/java/Dockerfile#L114

**Comment by kennethjor** [comment]
For the record, the answer is to add a `lookupName`:
```
# renovate: datasource=maven lookupName=org.apache.maven:maven
RUN install-tool maven 3.6.3
```
